.gitignore

```diff
@@ -1,33 +1,75 @@
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio, WebStorm and Rider
+# Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
+
+# User-specific stuff
+.idea/**/workspace.xml
+.idea/**/tasks.xml
+.idea/**/usage.statistics.xml
+.idea/**/dictionaries
+.idea/**/shelf
+
+# Generated files
+.idea/**/contentModel.xml
+
+# Sensitive or high-churn files
+.idea/**/dataSources/
+.idea/**/dataSources.ids
+.idea/**/dataSources.local.xml
+.idea/**/sqlDataSources.xml
+.idea/**/dynamic.xml
+.idea/**/uiDesigner.xml
+.idea/**/dbnavigator.xml
+
+# Gradle
+.idea/**/gradle.xml
+.idea/**/libraries
+
+# Gradle and Maven with auto-import
+# When using Gradle or Maven with auto-import, you should exclude module files,
+# since they will be recreated, and may cause churn.  Uncomment if using
+# auto-import.
+.idea/artifacts
+.idea/compiler.xml
+.idea/jarRepositories.xml
+.idea/modules.xml
+.idea/*.iml
+.idea/modules
 *.iml
-.idea/
+*.ipr
 
-target/
+# CMake
+cmake-build-*/
+
+# Mongo Explorer plugin
+.idea/**/mongoSettings.xml
+
+# File-based project format
+*.iws
+
+# IntelliJ
 out/
-output/
```

Comment:
Should `output` still be ignored? IIRC that's the cache where txt files got written after extraction - although that could certainly change with the new architecture.

Reply:
> I'm not sure I understand this. What information is in the `input` directory that needs to be versioned?
This directory is called `core/src/main/java/org/mobilitydata/gtfsvalidator/input/` and it contains Java files.

The original .gitignore file says that any directory called `input` should be ignored by git, even if it is a nested directory. This way I have lost all the Java files that I wrote and I was very happy to write them again :) So, let's not put such common names to .gitignore - we risk to discard hours of someone's work.

If we have a `ouptut` directory at the top level of the repo, we should write `/output` instead of just `output`. However, the new annotation-based validator will not create `output` directory.

Reply:
@aababilov Yikes! Not good. I agree with you - so this can stay as-is.